### DISCUSSION TITLE
After submitting the education certification application, I have not received a reply email for about ten days.

### DISCUSSION THREAD

**Discussion by Jerry-Xcq** [community/community#142272] [GitHub Education]
### Hi everyone,

About ten days ago, I applied for educational certification and did not receive any problem emails within one hour. However, to this day, I still have not received a reply email confirming successful certification. May I ask what stage we are currently in and how long it will take to receive email recovery? Thank you for your reply.

**Comment by davevad93** [comment]
Hi @Jerry-Xcq ,

You can check the status of your application [here](https://education.github.com/discount_requests/application), scroll down where it says "**You've already submitted X requests**"

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Jerry-Xcq was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
